tests/integration/workspace_access/test_generic.py

```diff
@@ -26,7 +26,7 @@
 
 
 @pytest.mark.parametrize("is_experimental", [True, False])
-@retried(on=[NotFound], timeout=timedelta(minutes=3))
+@retried(on=[NotFound, TimeoutError], timeout=timedelta(minutes=3))
```

Comment:
For which timeout is this intended? This is looks like a way to extend the timeout, which would like to avoid